### ISSUE TITLE
Pop-up window focus causes unintentional deletes

### ISSUE THREAD

**Issue by adegeofroy21** [phw198/OutlookGoogleCalendarSync#1193]
:memo: Use the below template to report your issue.  
:bulb: Upgrade to the latest alpha release to check if the problem is already fixed.  
:paperclip: To aid investigation, provide your OGcalsync.log file ([see instructions](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file)).  

----

**OGCS Version**: v2.8.6.0  
**Installed or Portable**: Installed

**Problem Description**  
\I have 'Confirm deletions' enabled, and recently OGCS has been popping up in front of all other applications whenever there is a confirmation pop-up. As a result, three times in 10 minutes I accidentally deleted calendar events that shouldn't have been because I was in the middle of typing an email when the pop-up appeared. I'd recommend making the pop-ups default to the background, and do not automatically put the focus on the confirmation button.

**Steps to Reproduce the Issue**  
1. \Be running OGC and typing something when the program requests deletion confirmation.
1. 

**Any other information**  
\<Update here>

**Comment by phw198** [comment]
If it pops up in the background, a greater number of user's would complain they didn't notice it and no syncs were happening.

:package: I've there opted to set the default focus on the "No" button - please try the [development build v2.8.6.9.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/6332009/v2.8.6.9.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix) in the same way as a hotfix).

Please let me know how it goes.

Note, deletion confirmations are really intended for initial testing of OGCS and not to always be left on - if you constantly get these popups it's an indication something isn't right. To resolve them as a one off, [follow this wiki page](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Hints-&-Tips#fixing-a-stubborn-unwanted-deletion-with-full-sync); if new occurrences keep appearing then this likely needs me to investigate what might be causing it.

**Comment by adegeofroy21** [comment]
Thanks for your response Paul. I haven't tried the new version yet, but
will shortly.

I have previously tried the troubleshooting steps you've linked to in the
wiki. Part of the reason I have 'Confirm deletions' permanently turned on
is because OGCS frequently attempts to delete items I've just created in
Outlook. Often I will create an item in Outlook and almost immediately I'll
have a popup asking me if I want to delete the item I just created. Perhaps
it's a timing issue? Is it possible that I'm creating these items in the
gap between the two unidirectional syncs? Does OGCS pull all items from
both calendars and merge to find differences, or does it only do two
unidirectional syncs back to back?
